sources/packages/web/src/views/aest/student/StudentAccountApplicationsApproval.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Accounts"
+        subTitle="View request"
+        :routeLocation="pendingAccountsRoute"
+      >
+        <template #buttons>
+          <v-row class="p-0 m-0">
+            <v-btn
+              color="primary"
+              class="mr-2"
+              variant="outlined"
+              @click="declineStudentAccount"
+              >Deny request</v-btn
+            >
+            <v-btn color="primary" outline @click="createStudentAccount"
```

Comment:
I think it's `variant="outlined"`. `d` at the end?

Reply:
Changed.